python/pyspark/errors/error_classes.py

```diff
@@ -1103,9 +1103,9 @@
       "Value for `<arg_name>` must be True, got '<arg_value>'."
     ]
   },
-  "VALUE_OUT_OF_BOUND": {
```

Comment:
my only concern is this error class name but I guess it's fine as we're bumping up the major release version this time.